### ISSUE TITLE
installing-repositories-to-galaxy is duplicated

### ISSUE THREAD

**Issue by nsoranzo** [galaxyproject/galaxy-hub#226]
Mostly exact duplication:
- https://galaxyproject.org/installing-repositories-to-galaxy/ (sources at `src/installing-repositories-to-galaxy/`)
- https://galaxyproject.org/toolshed/installing-repositories-to-galaxy/ (sources at `src/toolshed/installing-repositories-to-galaxy/`)

**Comment by martenson** [comment]
I would argue both of these are superseded by https://galaxyproject.org/admin/tools/add-tool-from-toolshed-tutorial/

**Comment by dannon** [comment]
@martenson I agree, https://galaxyproject.org/admin/tools/add-tool-from-toolshed-tutorial/ is a better endpoint here.  Do you definitely prefer the other pages archived, or can I delete them and add redirects?

**Comment by martenson** [comment]
@dannon I have treated archive differently so far -  redirects in place of archived URLs and the old text in archive so it is indexed and searchable.
